tests/test_journal_request.py

```diff
@@ -121,18 +124,120 @@ def test_journal_recruitment(self, openreview_client, selenium, request_page, he
+
+        messages = openreview_client.get_messages(to = 'ae_journal1@mail.com')
+        assert len(messages) == 0
+
+        messages = openreview_client.get_messages(to = 'ae_journal2@mail.com', subject = '[TJ22] Invitation to serve as action editor')
+        assert len(messages) == 1
+        assert messages[0]['content']['text'].startswith('<p>Dear Second AE,</p>\n<p>You have been nominated by the program chair committee of TJ22 to serve as action editor.</p>')
+
+        messages = openreview_client.get_messages(to = 'ae_journal3@mail.com', subject = '[TJ22] Invitation to serve as action editor')
+        assert len(messages) == 1
+        assert messages[0]['content']['text'].startswith('<p>Dear Third AE,</p>\n<p>You have been nominated by the program chair committee of TJ22 to serve as action editor.</p>')
+
+        inv = '{}/Journal_Request{}/-/Comment'.format(journal['suppot_group_id'],journal['journal_request_note']['number'])
+        recruitment_status = openreview_client.get_notes(invitation=inv, replyto=recruitment_note['note']['id'])
+
+        assert recruitment_status
+        assert recruitment_status[0].content['title']['value'] == 'Recruitment Status'
+        assert 'Invited: 2 action editors.' in recruitment_status[0].content['comment']['value']
+
+    def test_journal_reviewer_recruitment_by_ae(self, openreview_client, selenium, request_page, helpers, journal):
+
+        #add ae to action editors group
+        openreview_client.add_members_to_group(journal['journal_request_note']['content']['venue_id']['value']+ '/Action_Editors', 'ae_journal1@mail.com')
+
+        helpers.create_user('ae_journal1@mail.com', 'First', 'AE')
+        ae_client = OpenReviewClient(username='ae_journal1@mail.com', password='1234')
+
+        request_page(selenium, 'http://localhost:3030/forum?id={}'.format(journal['journal_request_note']['id']), ae_client.token)
+        recruitment_div = selenium.find_element_by_id('note_{}'.format(journal['journal_request_note']['id']))
+        assert recruitment_div
+        reply_row = recruitment_div.find_element_by_class_name('reply_row')
+        assert reply_row
+        buttons = reply_row.find_elements_by_class_name('btn-xs')
+        assert [btn for btn in buttons if btn.text == 'Reviewer Recruitment']
+
+        reviewer_details = { 'value': '''new_reviewer@mail.com, New Reviewer'''}
+        recruitment_note = ae_client.post_note_edit(
+            invitation = '{}/Journal_Request{}/-/Reviewer_Recruitment'.format(journal['suppot_group_id'],journal['journal_request_note']['number']),
+            signatures = ['~First_AE1'],
+            note = Note(
+                content = {
+                    'invitee_details': reviewer_details,
+                    'email_subject': { 'value': '[' + journal['journal_request_note']['content']['abbreviated_venue_name']['value'] + '] Invitation to serve as reviewer' },
```

Comment:
I wonder if we should allow the AE to edit the subject and content.

Reply:
or may be we should add to the template a variable that show the inviter, the tilde id/email of the AE.